spec/ruby_llm/mcp/parameter_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe RubyLLM::MCP::Parameter do
+  describe "#item_type" do
+    context "when @items is nil" do
```

Comment:
These context block was just on it statments - can you collapse them down or bucket some it statements together